strictdoc/core/actions/import_action.py

```diff
@@ -0,0 +1,30 @@
+import os
+from pathlib import Path
+
+from strictdoc.backend.excel.excel_import import ExcelImport
+from strictdoc.backend.reqif.reqif_import import ReqIFImport
+from strictdoc.backend.sdoc.writer import SDWriter
+from strictdoc.cli.cli_arg_parser import (
+    ImportCommandConfig,
+)
+from strictdoc.helpers.timing import timing_decorator
+
+
+class ImportAction:
+    @staticmethod
+    @timing_decorator("Import")
+    def do_import(import_config: ImportCommandConfig):
```

Comment:
Please add a type annotation for the return type.

Reply:
-> None ? Done